website/source/docs/guides/acl.html.md

```diff
@@ -218,6 +218,21 @@ curl http://127.0.0.1:8500/v1/catalog/nodes -H 'x-consul-token: 4411f091-a4c9-48
 
 All the values should be as expected. Particularly, if `TaggedAddresses` is `null` it is likely we have not configured ACLs correctly. A good place to start debugging is reviewing the Consul logs on all the servers.
 
+If you encounter issues that are unresolvable, or misplace the bootstrap token, you can reset the ACL system by updating the index. First re-run the bootstrap command to get the index number.
+
+```
+$ consul acl bootstrap
+Error bootstrapping: Unexpected response code: 500 (ACL bootstrap already done (reset index: 13))
+```
+
+Then update the index with the following command.
```

Comment:
```suggestion
Then write the reset index into the bootstrap reset file: (here the reset index is 13)
```

Maybe something more like this would be better. Basically what you are doing is writing the reset index you got from the first call into the bootstrap reset file to indicate to Consul that it should allow bootstrapping again.

Reply:
LGTM. I like that you were more specific, no room for questions :)